service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/PolicyEvaluationRequestCacheKey.java

```diff
@@ -100,8 +96,8 @@ public PolicyEvaluationRequestV1 getRequest() {
         return this.request;
     }
 
-    public String getPolicySetId() {
-        return this.policySetId;
+    public List<String> getPolicySetIds() {
+        return this.policySetIds;
```

Comment:
Once again, shouldn't the collection of policy set ids be a set instead of a list?